### PR TITLE
0.2.2 release

### PR DESCRIPTION
**Pull request by pandafy**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,26 @@
 Changelog
 =========
 
+Version 0.2.2 [2020-11-19]
```
Comment (nemesisdesign) [inline CHANGES.rst:4]: date will need to be updated

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+
+- [dependencies] Upgraded ``openwisp-utils~=0.7.0`` and
+  ``openwisp-users~=0.5.0``
+
+Features
+~~~~~~~~
+
+- Added management command to populate notification preferences
+
+
```
Comment (nemesisdesign) [inline CHANGES.rst:23]: I'd remove this extra space

File: CHANGES.rst [hunk 3]
```diff
@@ -1,6 +1,26 @@
 Changelog
 =========
 
+Version 0.2.2 [2020-11-19]
+--------------------------
+
+Bugfixes
+~~~~~~~~
+
+- Fixed notification alert sound being played from multiple windows
```
Comment (nemesisdesign) [inline CHANGES.rst:10]: looks like this is missing 2e4b39cb4b1b7b20ee11ec2158e451e715aa2f03

File: CHANGES.rst [hunk 4]
```diff
@@ -1,6 +1,26 @@
...
+Changes
+~~~~~~~
+
+- [dependencies] Upgraded ``openwisp-utils~=0.7.0`` and
+  ``openwisp-users~=0.5.0``
+
+Features
+~~~~~~~~
+
+- Added management command to populate notification preferences
```
Comment (nemesisdesign) [inline CHANGES.rst:21]: if there's a new feature, we should increase the version number to 0.3

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Once the issues below are resolved, please go ahead and issue the release.

File: CHANGES.rst (hunk 3 above)
Comment (pandafy) [reply CHANGES.rst:10]: This fix only works for the test project. It does not applies to module due to the bug mentioned in the PR.

**Pull request merged**